src/context.cpp

```diff
@@ -190,7 +190,7 @@ namespace librealsense
             
             if( _dds_watcher && _dds_watcher->is_stopped() )
             {
-                start_dds_device_watcher();
+                start_dds_device_watcher( utilities::json::get< size_t >( settings, "dds-message-timeout-ms", 5000 ) );
```

Comment:
The timeout is for every DDS messages?

Reply:
Not sure this is the right place to transfer this value.
This function is for activating a DDS device watcher, it should have parameters about the operation mode of the DDS device IMO

Reply:
Currently this timeout is used only when setting and getting options, but this implies the expected round trip time for all messages. We will have to test it later with actual remote camera (not on same computer) I expect it to be less than 5 seconds.

Reply:
Do we have another entry that is more relevant to use this settings?

Reply:
Don't understand your question